src/middlewared/middlewared/api/v25_04_0/virt.py

```diff
@@ -96,9 +96,14 @@ class Disk(Device):
     destination: Optional[str] = None
 
 
+NicType: TypeAlias = Literal['BRIDGED', 'MACVLAN']
+
+
 class NIC(Device):
     dev_type: Literal['NIC']
-    network: NonEmptyString
+    network: Optional[NonEmptyString] = None
```

Comment:
This file has a mixture of
`value1 | value2` and `Optional[value1] = None`. Please change it so that we remove the `Optional` builtin since that has actually been deprecated by python already.

```
network: NonEmptyString | None = None